selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -376,4 +385,29 @@ private void checkExpectedJavaWorkspaceDisplaying() {
   private int getWorkspacesCount() throws Exception {
     return testWorkspaceServiceClient.getAll().size();
   }
+
+  private void waitOfOpositeCheckboxState(String workspaceName, boolean currentCheckboxState) {
+    if (currentCheckboxState) {
+      workspaces.waitWorkspaceCheckboxDisabled(workspaceName);
+      return;
+    }
+
+    workspaces.waitWorkspaceCheckboxEnabled(workspaceName);
+  }
+
+  private void clickOnUnexpectedWorkspacesCheckboxes(List<String> expectedWorkspaces) {
```

Comment:
This method would be useful for other tests as well. How about moving it into the **Workspaces** page object as well?

Reply:
Also, it's not quite clear how *Unexpected* workspaces differ from ordinary one.